src/redturtle/prenotazioni/tests/test_add_booking.py

```diff
@@ -80,13 +81,23 @@ def setUp(self):
             title="Prenota foo",
             description="",
             daData=date.today(),
-            booking_types=[
-                {"name": "Type A", "duration": "30"},
-            ],
             gates=["Gate A", "Gate B"],
             week_table=WEEK_TABLE_SCHEMA,
         )
+
+        booking_type_A = api.content.create(
+            type="BookingType",
+            title="Type A",
+            duration=30,
+            container=self.folder_prenotazioni,
+            gates=["all"],
+        )
+
         api.content.transition(obj=self.folder_prenotazioni, transition="publish")
+        api.content.transition(obj=booking_type_A, transition="publish")
+        self.folder_prenotazioni.reindexObject(idxs="review_state")
```

Comment:
you shouldn't need to reindex object here. Are you sure?

Reply:
no, thnx, done